lib/ansible/module_utils/postgres.py

```diff
@@ -29,12 +29,14 @@
 
 try:
     import psycopg2
-    import psycopg2.extras
+    from psycopg2.extras import DictCursor
     HAS_PSYCOPG2 = True
```

Comment:
Any reason to not keep `PSYCOPG2_IMP_ERR = traceback.format_exc()` ? This was [recently added](https://github.com/ansible/ansible/pull/51787/files#diff-fd1114643a7c9394b36aac9044e2c581) ?

Reply:
@pilou- , I remember that seems @dagwieers said traceback is no needed anymore in this way because of it's added automatically.